pkg/resource/endpoint/custom_set_output.go

```diff
@@ -47,6 +47,19 @@ func (rm *resourceManager) customDescribeEndpointSetOutput(
 	ko *svcapitypes.Endpoint,
 ) (*svcapitypes.Endpoint, error) {
 	rm.customSetOutput(r, resp.EndpointStatus, ko)
+	// Workaround: Field config for LatestEndpointConfigName of generator config
```

Comment:
major nit: Add punctuation to comments, took me a couple of rereads to realize these were different sentences.

Reply:
will do in a future PR